lib/ability.rb

```diff
@@ -126,6 +128,8 @@ def initialize(user, resource, controller = nil)
         end
 
         can [:administer], User
+        can([:edit, :update], User, &:external?)
```

Comment:
This is an open question we never really discussed. Who should be allowed to edit name/emails? I'm leaning towards only global admins. Otherwise, I think it opens the possibility of someone messing something up inadvertently. If we do want to open it up to more people, some sort of audit trail would be a good idea.